### PR TITLE
Fix report

### PR DESCRIPTION
**Pull request by githinho**

### CODE REVIEW

File: src/main/resources/templates/style.css [hunk 1]
```diff
@@ -79,8 +79,13 @@ table td {
     padding-bottom: 8px;
 }
 
-.page-1 .account-summary .summary-info .summary-right {
-    margin-top: 42px;
+/*.summary-right .summary-left {*/
+/*    position: absolute;*/
+/*    bottom: 0;*/
+/*}*/
```
Comment (mightymatth) [inline src/main/resources/templates/style.css:85]: Remove comments.

**Review by mightymatth — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/user-transactions-template.html [hunk 2]
```diff
@@ -106,7 +105,7 @@ <h2>Transactions</h2>
                     <th class="table-border-right">Pending</th>
                     <th>Paid Out</th>
                 </tr>
-                <tr th:each="tx,iStat : ${txSummary.transactions}" th:classappend="${(iStat.even? 'table-grey-background': '') + (iStat.last? 'table-border-bottom': '')}">
+                <tr th:each="tx,iStat : ${txSummary.transactions}" th:classappend="${(iStat.even? 'table-grey-background': '') + ' ' + (iStat.last? 'table-border-bottom': '')}">
```
Comment (josipkoncurat) [inline src/main/resources/templates/user-transactions-template.html:108]: adding + ' ' + doesn't fix the issue table-border-bottom on last transaction

**Review by mightymatth — APPROVED** [review]
Incredible!

**Review by josipkoncurat — APPROVED** [review]

File: src/main/resources/templates/user-transactions-template.html (hunk 2 above)
Comment (githinho) [reply src/main/resources/templates/user-transactions-template.html:108]: Without space, the last even row wasn't gray.

**Pull request merged**